tests/gdx-tests/src/com/badlogic/gdx/tests/VBOWithVAOPerformanceTest.java

```diff
@@ -0,0 +1,491 @@
+		bitmapFont = new BitmapFont();
+		stringBuilder = new StringBuilder();
+	}
+
+	private void addRandomSprite (float[] vertArray, int currentIndex) {
+		float width = MathUtils.random(0.05f, 0.2f);
+		float height = MathUtils.random(0.05f, 0.2f);
+		float x = MathUtils.random(-1f, 1f);
+		float y = MathUtils.random(-1f, 1f);
+		float r = MathUtils.random();
+		float g = MathUtils.random();
+		float b = MathUtils.random();
+		float a = MathUtils.random();
+
+		vertArray[currentIndex++] = x;
+		vertArray[currentIndex++] = y;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = r;
+		vertArray[currentIndex++] = g;
+		vertArray[currentIndex++] = b;
+		vertArray[currentIndex++] = a;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = 1;
+
+		vertArray[currentIndex++] = x + width;
+		vertArray[currentIndex++] = y;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = r;
+		vertArray[currentIndex++] = g;
+		vertArray[currentIndex++] = b;
+		vertArray[currentIndex++] = a;
+		vertArray[currentIndex++] = 1;
+		vertArray[currentIndex++] = 1;
+
+		vertArray[currentIndex++] = x + width;
+		vertArray[currentIndex++] = y + height;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = r;
+		vertArray[currentIndex++] = g;
+		vertArray[currentIndex++] = b;
+		vertArray[currentIndex++] = a;
+		vertArray[currentIndex++] = 1;
+		vertArray[currentIndex++] = 0;
+
+		vertArray[currentIndex++] = x + width;
+		vertArray[currentIndex++] = y + height;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = r;
+		vertArray[currentIndex++] = g;
+		vertArray[currentIndex++] = b;
+		vertArray[currentIndex++] = a;
+		vertArray[currentIndex++] = 1;
+		vertArray[currentIndex++] = 0;
+
+		vertArray[currentIndex++] = x;
+		vertArray[currentIndex++] = y + height;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = r;
+		vertArray[currentIndex++] = g;
+		vertArray[currentIndex++] = b;
+		vertArray[currentIndex++] = a;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = 0;
+
+		vertArray[currentIndex++] = x;
+		vertArray[currentIndex++] = y;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = r;
+		vertArray[currentIndex++] = g;
+		vertArray[currentIndex++] = b;
+		vertArray[currentIndex++] = a;
+		vertArray[currentIndex++] = 0;
+		vertArray[currentIndex++] = 1;
+	}
+
+	@Override
+	public void render () {
+		Gdx.gl20.glViewport(0, 0, Gdx.graphics.getBackBufferWidth(), Gdx.graphics.getBackBufferHeight());
+		Gdx.gl20.glClearColor(0.2f, 0.2f, 0.2f, 1);
+		Gdx.gl20.glClear(GL20.GL_COLOR_BUFFER_BIT);
+		Gdx.gl20.glEnable(GL20.GL_TEXTURE_2D);
+		Gdx.gl20.glEnable(GL20.GL_BLEND);
+		Gdx.gl20.glBlendFunc(GL20.GL_SRC_ALPHA, GL20.GL_ONE_MINUS_SRC_ALPHA);
+
+
+		texture.bind();
+		shader.begin();
+		shader.setUniformMatrix("u_worldView", matrix);
+		shader.setUniformi("u_texture", 0);
+
+		long beforeOld = System.nanoTime();
+		oldVBOWithVAOMesh.render(shader, GL20.GL_TRIANGLES);
```

Comment:
Since these calls are asynchronous, better call `glFlush` to make sure it finished.